src/components/modals/RadioListModal.tsx

```diff
@@ -20,37 +29,59 @@ export function RadioListModal(props: Props) {
   const theme = useTheme()
   const styles = getStyles(theme)
 
-  function renderRow(item: string): React.ReactNode {
-    const radio = {
-      icon: `ios-radio-button-${selected === item ? 'on' : 'off'}`,
-      color: theme.iconTappable
-    }
+  const renderRow = useHandler((item: Item) => {
+    const { name, icon, text } = item
+
+    const isSelected = selected === name
+    const radio = isSelected
+      ? { icon: 'ios-radio-button-on', color: theme.iconTappable }
+      : { icon: 'ios-radio-button-off', color: theme.iconTappable }
+    const accessibilityState = isSelected
+      ? { checked: true }
+      : { checked: false }
+    const accessibilityHint = `${
```

Comment:
I don't see any changes here, aside from s.strings / lstrings. Would it make sense to port the login-ui to use "lstrings" like the GUI does?